policy/policy.go

```diff
@@ -139,11 +125,110 @@ func ParsePolicy(str string) (Policy, error) {
 	return Policy{resolution: resolution, retention: retention}, nil
 }
 
-// NewPoliciesFromSchema creates multiple new policues from given schema policies.
-func NewPoliciesFromSchema(policies []*schema.Policy) ([]Policy, error) {
-	res := make([]Policy, 0, len(policies))
+// UnaggregatedPolicy represents a policy with a list of custom aggregation types
+type UnaggregatedPolicy struct {
```

Comment:
Would it be better to create an `unaggregated` package under `policy` and put the `UnaggregatedPolicy` in there? This way we can refer to it as `unaggregated.Policy` which might be slightly more clear, just like how we structure the `metrics` package.